core/driver/sources/src/main/resources/reference.conf

```diff
@@ -79,6 +79,16 @@ probe {
     //
     // If you want to use the Ultimate edition, replace "ideaIC" occurences with "ideaIU" in your config file. Community
     // edition ("ideaIC") is the default one.
+    //
+    // You can also add your custom intellij repository on top of the 4 default repositories, so that ide-probe will
+    // first try using the IntelliJs downloaded from your repository if present. If ide-probe doesn't find the IntelliJ
+    // distribution in your repository, it will use the default 4 repositories listed below. To achieve that, specify
+    // your custom repository pattern and add the `official` or `default` keyword on another line. Example below:
+    // intellij.repositories = [
+    //   "file:///"${HOME}"/.my_intellij_idea_downloads/[revision]/ideaIC-[revision][format]",
```

Comment:
If it makes sense, you can actually use `.pluginVerifier` as an example (with a link to https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier)